### PR TITLE
Edits to cli guide

### PR DESCRIPTION
**Pull request by atravitz**
Edits/suggestions from me reading through these docs as a new user.

## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT License [here](https://github.com/OpenFreeEnergy/openfe/blob/main/LICENSE) and the **Developer Certificate of Origin** at <https://developercertificate.org/>.

### CODE REVIEW

File: docs/guide/cli/cli_yaml.rst [hunk 1]
```diff
@@ -2,9 +2,10 @@ Customising CLI planning with yaml settings
 ===========================================
 
 The planning commands in the CLI can be made more powerful by supplying
-"yaml" formatted files to customise the planning algorithms.
-This settings file has a series of sections for customising the different algorithms,
-as an example, the settings file which re-specifies the default behaviour would look like ::
+yaml-formatted files to customise the planning algorithms.
```
Comment (IAlibay) [inline docs/guide/cli/cli_yaml.rst:5]:
```suggestion
YAML-formatted files to customise the planning algorithms.
```
I believe it's meant to be capitalized so we probably should do that when we talk about the "thing".

File: docs/guide/cli/cli_yaml.rst [hunk 2]
```diff
@@ -2,9 +2,10 @@ Customising CLI planning with yaml settings
...
+
+This settings file has a series of sections for customising the different algorithms.
```
Comment (IAlibay) [inline docs/guide/cli/cli_yaml.rst:7]:
```suggestion
This settings file has a series of sections for customizing the different algorithms.
```
Historical british-isms are probably everywhere.

File: docs/guide/cli/cli_yaml.rst [hunk 3]
```diff
@@ -18,18 +19,21 @@ as an example, the settings file which re-specifies the default behaviour would
 
 The name of the algorithm is given behind the ``method:`` key and the arguments to the
 algorithm are then optionally given behind the ``settings:`` key.
-Both the `network:` and `mapper:` sections are optional.
+Both the ``network:`` and ``mapper:`` sections are optional.
 
-This is then provided to the ``openfe plan-rbfe-network`` command as ::
+The settings yaml file is then provided to the ``-s`` option of ``openfe plan-rbfe-network``: ::
```
Comment (IAlibay) [inline docs/guide/cli/cli_yaml.rst:24]:
```suggestion
The settings YAML file is then provided to the ``-s`` option of ``openfe plan-rbfe-network``: ::
```
etc.. please reject if it's too much of a bother.

File: docs/guide/cli/cli_yaml.rst [hunk 4]
```diff
@@ -18,18 +19,21 @@ as an example, the settings file which re-specifies the default behaviour would
...
 ---------------------------
 
 There is a choice to be made as to which atom mapper is used,
-currently included are the :class:`.LomapAtomMapper` and the :class:`.KartografAtomMapper`
-For example to switch to using the ``Kartograf`` atom mapper, this settings yaml could be used ::
+currently included are the :class:`.LomapAtomMapper` and the :class:`.KartografAtomMapper` (full details in the `Kartograf documentation`_.)
+
+.. _Kartograf documentation: https://kartograf.readthedocs.io/en/latest/api/kartograf.mappers.html#kartograf.atom_mapper.KartografAtomMapper
+
+For example, to switch to using the ``Kartograf`` atom mapper, this settings yaml could be used ::
```
Comment (IAlibay) [inline docs/guide/cli/cli_yaml.rst:36]:
```suggestion
For example, to switch to using the ``Kartograf`` atom mapper, this settings YAML could be used ::
```

**Review by IAlibay — APPROVED** [review]
lgtm thanks! just a couple of suggestions for issues prior to this PR that could be quickly addressed if we want to

File: docs/guide/cli/cli_yaml.rst (hunk 1 above)
Comment (atravitz) [reply docs/guide/cli/cli_yaml.rst:5]: agreed

**Pull request merged**